File: lib/src/main/java/com/sha/formvalidator/validator/pattern/IpAddressValidator.java
package com.sha.formvalidator.validator.pattern;

import android.util.Patterns;

/**
 * Validates the ipaddress. The regexp was taken from the android source code.
 *
 */
public class IpAddressValidator extends PatternValidator {
    public IpAddressValidator(String errorMessage) {
        super(errorMessage, Patterns.IP_ADDRESS);
    }
}

File: lib/src/main/java/com/sha/formvalidator/textview/validator/pattern/IpAddressValidator.kt
package com.sha.formvalidator.textview.validator.pattern

import android.util.Patterns

/**
 * Validates the ipaddress. The regexp was taken from the android source code.
 *
 */
class IpAddressValidator(errorMessage: String) : PatternValidator(errorMessage, Patterns.IP_ADDRESS)
